index.html

```diff
@@ -2,12 +2,190 @@
     <title>Main Page</title>
   </head>
   <body>
-  <h1>Моя первая страница</h1>
-  <p>
-    <a href="./about">Об авторе</a>
-  </p>
+    <h1>Моя первая страница</h1>
+    <p>
+      <a href="./about">Об авторе</a>
+    </p>
+    <hr />
+    <h2>Image</h2>
+    <img src="./images/white bear.webp" alt="Sitting white bear" />
+    <h2>List</h2>
+    <ol>
+      <li>
+        Знакомство с Web
+        <ul>
+          <li>Краткая история Web и как устроен интернет</li>
+          <li>Как устроен браузер и как он получает страницы по сети</li>
+          <li>Что такое Git</li>
+          <li>Создание первого репозитория</li>
+          <li>О файле .gitignore</li>
+        </ul>
+      </li>
+      <li>
+        HTML-вёрстка – первые шаги
+        <ul>
+          <li>Настройка рабочей среды</li>
+          <li>Краткая история HTML</li>
+          <li>Типы тегов и атрибуты</li>
+          <li>Теги для работы с текстом</li>
+          <li>Создание собственной страницы</li>
+          <li>Основы ветвления в Git</li>
+          <li>Создание pull request на GitHub</li>
+        </ul>
+      </li>
+      <li>
+        Погружение в HTML-разметку
+        <ul>
+          <li>Мета-теги</li>
+          <li>Мнемоники</li>
+          <li>Теги для работы со списками</li>
+          <li>Теги для работы с таблицами</li>
+          <li>Теги для работы с формами</li>
+          <li>p и span</li>
+        </ul>
+      </li>
+    </ol>
+    <h2>Table</h2>
+    <table border="1" width="25%">
+      <thead>
+        <tr>
+          <th align="center">Tag</th>
+          <th align="center">Description</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th align="left">&lt;table&gt;</th>
+          <th align="left">Defines a table</th>
+        </tr>
+        <tr>
+          <th align="left">&lt;tr&gt;</th>
+          <th align="left">Defines a row in a table</th>
+        </tr>
+        <tr>
+          <th align="left">&lt;th&gt;</th>
+          <th align="left" rowspan="2">
+            Defines a "header cell" / "cell" in a table
+          </th>
+        </tr>
+        <tr>
+          <th align="left">&lt;td&gt;</th>
+        </tr>
+        <tr>
+          <th align="left" colspan="2">Rarely used</th>
+        </tr>
+        <tr>
+          <th align="left">&lt;caption&gt;</th>
+          <th align="left">Defines a table caption</th>
+        </tr>
+        <tr>
+          <th align="left">&lt;thead&gt;</th>
+          <th align="left">Groups the header content in a table</th>
+        </tr>
+        <tr>
+          <th align="left">&lt;tbody&gt;</th>
+          <th align="left">Groups the body content in a table</th>
+        </tr>
+        <tr>
+          <th align="left">&lt;tfoot&gt;</th>
+          <th align="left">Groups the foot content in a table</th>
+        </tr>
+      </tbody>
+    </table>
+    <form action="/" method="get">
+      <fieldset>
+        <legend>My first form</legend>
+        <p>
```

Comment:
`<p></p>` is for the text paragraph.
Block should be wrapped into `<div></div>`